src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c

```diff
@@ -33,6 +33,12 @@ ActorInit Shot_Sun_InitVars = {
     NULL,
 };
 
+typedef enum {
+    SPAWNER_INACTIVE,
+    SPAWNER_INRANGE,
+    SPAWNER_OCARINA
+}FairySpawnerState;
```

Comment:
If you run the formatter on this, it'll end up like
```c
typedef enum { SPAWNER_INACTIVE, SPAWNER_INRANGE, SPAWNER_OCARINA } FairySpawnerState;
```
to avoid this we usually put `/* value */` comments for each enum name (and it's also vaguely useful for reference)
like:
```c
typedef enum {
    /* 0 */ SPAWNER_INACTIVE,
    /* 1 */ SPAWNER_INRANGE,
    /* 2 */ SPAWNER_OCARINA
} FairySpawnerState;
```

Also imo `SPAWNER_INRANGE` would be better as `SPAWNER_IN_RANGE`